src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/index.unit.spec.jsx

```diff
@@ -697,9 +699,18 @@ describe('VAOS <CommunityCareProviderSelectionPage>', () => {
         store,
       },
     );
-    await screen.findByText(/We’ll call you to confirm your provider choice/);
 
-    // Then the closest city/state question is not shown
+    // Then the heading will display type of provider requested
+    expect(
+      await screen.findByRole('heading', {
+        level: 1,
+        name: /Request a Primary care provider/i,
+      }),
+    ).to.be.ok;
+    expect(screen.getByText(/We’ll call you to confirm your provider choice/))
```

Comment:
It looks like this fails sometimes (it happened in Jenkins and I saw it happen once locally). Might be worth doing `expect(await screen.findByText(/We’ll call you to confirm your provider choice/)).to.be.ok;`